pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1312,7 +1314,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			for feature, value := range tc.featureGates {
 				defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, feature, value)()
 			}
-			defer algorithmprovider.ApplyFeatureGates()()
```

Comment:
Let's also cleanup other occurrences in `test/integration`:

```bash
⇒  ag "ApplyFeatureGates" test/integration/scheduler
test/integration/scheduler/priorities_test.go
251:	defer algorithmprovider.ApplyFeatureGates()()

test/integration/scheduler/taint_test.go
87:	defer algorithmprovider.ApplyFeatureGates()()
651:	// ApplyFeatureGates() is called to ensure TaintNodesByCondition related logic is applied/restored properly.
652:	defer algorithmprovider.ApplyFeatureGates()()

test/integration/scheduler/predicates_test.go
883:	defer algorithmprovider.ApplyFeatureGates()()
```

Reply:
done.